src/resources/MachineLearning/SESConfiguration/SESConfiguration.ts

```diff
@@ -0,0 +1,12 @@
+import API from '../../../APICore.js';
+import Resource from '../../Resource.js';
+import {SESDocumentGroupPreviewParams, SESDocumentGroupPreview} from './SESConfigurationInterfaces.js';
+
+export default class SESConfiguration extends Resource {
```

Comment:
Should we go with the full name instead of SES? Or at least some jsdoc explaining what SES stands for. I personally have no clue what this means and probably so do our clients.

Reply:
I changed, it will be more meaningful